CONTRIBUTING.md

```diff
@@ -0,0 +1,298 @@
+```
+  - --input=data
+  - --mean_values=data[127.5]
+  - --scale_values=data[127.5]
+  - --reverse_input_channels
+  - --output=prob
+  - --input_model=$conv_dir/googlenet-v3.onnx
+```
+> **NOTE:** Do not specify `framework`, `data_type`, `model_name` and `output_dir`, since they are deduced automatically.
+
+**`framework`**
+
+Framework of the original model. Examples: `caffe`, `dldt`, `mxnet`, `pytorch`, `tf`.
+
+**`license`**
+
+Path to the model license.
+
+### Example
+
+This example shows how to download  the [classification model DenseNet-121*](models/public/densenet-121-tf/model.yml) pretrained in TensorFlow\*  from Google Drive\* as an archive.
+
+```
+description: >-
+  This is an Tensorflow\* version of `densenet-121` model, one of the DenseNet
+  group of models designed to perform image classification. The weights were converted
+  from DenseNet-Keras Models. For details see repository <https://github.com/pudae/tensorflow-densenet/>,
+  paper <https://arxiv.org/pdf/1608.06993.pdf>
+task_type: classification
+files:
+  - name: tf-densenet121.tar.gz
+    size: 30597420
+    sha256: b31ec840358f1d20e1c6364d05ce463cb0bc0480042e663ad54547189501852d
+    source:
+      $type: google_drive
+      id: 0B_fUSpodN0t0eW1sVk1aeWREaDA
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: tf-densenet121.tar.gz
+model_optimizer_args:
+  - --reverse_input_channels
+  - --input_shape=[1,224,224,3]
+  - --input=Placeholder
+  - --mean_values=Placeholder[123.68,116.78,103.94]
+  - --scale_values=Placeholder[58.8235294117647]
+  - --output=densenet121/predictions/Reshape_1
+  - --input_meta_graph=$dl_dir/tf-densenet121.ckpt.meta
+framework: tf
+license: https://raw.githubusercontent.com/pudae/tensorflow-densenet/master/LICENSE
+```
+
+## Model Conversion
+
+Deep Learning Inference Engine (IE) supports models in the Intermediate Representation (IR) format. A model from any supported framework can be converted to IR using the Model Optimizer tool included in the OpenVINO&trade; toolkit. Find more information about conversion in the [Model Optimizer Developer Guide](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After a successful conversion you get a model in the IR format, with the `*.xml` file representing the net graph and the `*.bin` file containing the net parameters. 
+
+> **NOTE 1**: Image preprocessing parameters (mean and scale) must be built into a converted model to simplify model usage.
+
+> **NOTE 2**: If a model input is a color image, color channel order should be `BGR`.
+
+## Demo
+
+A demo shows the main idea of how to infer a model using IE. If your model solves one of the tasks supported by the Open Model Zoo, try to find an appropriate option from [demos](https://docs.openvinotoolkit.org/latest/_demos_README.html) or [samples](https://docs.openvinotoolkit.org/latest/_docs_IE_DG_Samples_Overview.html). Otherwise, you must provide your own demo (C++ or Python). 
```

Comment:
The `demos` link should be a relative link to the file in this repository, since that's going to have more up-to-date information.